api/_mixins/HTMLHyperlinkElementUtils__HTMLAnchorElement.json

```diff
@@ -24,7 +24,7 @@
               "notes": "From Firefox 29 to Firefox 40, the returned value was incorrectly percent-decoded."
             },
             "ie": {
-              "version_added": "5"
+              "version_added": "5.5"
```

Comment:
All of the IE 5.5 changes make sense if we trust the api.HTMLElement data, which says IE 5.5. I don't really trust that data, but let's not spend any time verifying pre-IE6 data that the collector can't test.